eng/CapturedBuildOutputCrossTargetingTargets.targets

```diff
@@ -0,0 +1,6 @@
+<!-- Overriding the common Build target to capture the inner outputs: https://github.com/microsoft/msbuild/issues/2148. -->
```

Comment:
It looks like this issue is closed, should we reopen it?

Reply:
Asked @rainersigwald to reopen: https://github.com/microsoft/msbuild/issues/2148